rpcserver.go

```diff
@@ -172,6 +172,7 @@ var rpcHandlersBeforeInit = map[string]commandHandler{
 	"validateaddress":       handleValidateAddress,
 	"verifychain":           handleVerifyChain,
 	"verifymessage":         handleVerifyMessage,
+	"estimatefee":           handleEstimateFee,
```

Comment:
Please alphabetize this to be consistent with the rest of the handlers.  After `decdeScript`.